src/entries/popup/components/AddressDisplay.tsx

```diff
@@ -47,6 +54,18 @@ function AddressMoreOptions({ address }: { address: Address }) {
             {truncateAddress(address)}
           </Text>
         </DropdownMenuItem>
+        {etherscanLink && (
+          <DropdownMenuItem
+            symbolLeft="doc.text.magnifyingglass"
+            onSelect={() =>
+              goToNewTab({ url: `https://etherscan.io/address/${address}` })
+            }
```

Comment:
To support other explorer, we'll want to make this generic. We get explorer URLs upstream from `viem/chains` and you can reference this from the Activity right-click https://github.com/rainbow-me/browser-extension/blob/35e3739098267fab82e67f896719befd156caca1/src/entries/popup/pages/home/Activity/ActivityContextMenu.tsx#L46